README.md

```diff
@@ -16,11 +16,11 @@
 
 The Pi-hole® is a [DNS sinkhole](https://en.wikipedia.org/wiki/DNS_Sinkhole) that protects your devices from unwanted content without installing any client-side software.
 
-- **Easy-to-install**: our versatile installer walks you through the process and takes less than ten minutes
+- **Easy-to-install**: our versatile installer walks you through the installation process that takes less than ten minutes
 - **Resolute**: content is blocked in _non-browser locations_, such as ad-laden mobile apps and smart TVs
 - **Responsive**: seamlessly speeds up the feel of everyday browsing by caching DNS queries
 - **Lightweight**: runs smoothly with [minimal hardware and software requirements](https://docs.pi-hole.net/main/prerequisites/)
-- **Robust**: a command line interface that is quality assured for interoperability
+- **Robust**: a command-line interface that is quality assured for interoperability
```

Comment:
I think this is correct?

https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/c/command-line

Reply:
"command line" is an adjective in this case and should be hyphenated as an adjective, according to the guideline you've linked.

Reply:
https://en.wiktionary.org/wiki/command_line_interface

Maybe both are correct, but `command-line` seems to be more generally accepted.